docker/dropin_env_base_jdk_ubi/build_image.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Copyright 2021 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+IMAGE_NAME=datarobot/dropin-env-base-jdk
+IMAGE_TAG=ubi8.8-py3.11-jdk11.0.20-drum1.10.9-mlops9.1.3
```

Comment:
No for this PR, but maybe consider having all the related versions in a single file and read the versions in all required places. This way, you'll always make a change in one place only.

Reply:
ok